jest.config.base.js

```diff
@@ -83,7 +83,7 @@ module.exports = {
   moduleNameMapper: null,
 
   // An array of regexp pattern strings, matched against all module paths before considered 'visible' to the module loader
-  modulePathIgnorePatterns: ['<rootDir>/.build', '<rootDir>/dist/'],
+  modulePathIgnorePatterns: ['<rootDir>/.build', '<rootDir>/dist/', '<rootDir>/build/'],
```

Comment:
Currently, only airnode-validator uses the build folder, but I've put it to the root jest file because:
- we should not ever have tests in "build" folder
- was the easy choice

Validator needs to us the "build" folder, because projects which are referenced need to be set `composite` to `true`. This setting also implies that `noEmit` cannot be set to `true` and TS must be allowed to output the build files somewhere.